### DISCUSSION TITLE
Set Time and Date to automatically Promote Achievements to Core and Display Release Time

### DISCUSSION THREAD

**Discussion by ghost** [RetroAchievements/RAWeb#790] [Ideas]
@MrOwnership recently added this cool new feature to the Site which allows to promote many selected Achievements to Core at once. https://github.com/RetroAchievements/RAWeb/pull/420

This could be expanded into a automatically Release Uploader/Planer with a indicactor on each Game Page when the Achievements will go online. This info could also be displayed on the Front Page under 'Upcoming Sets' with the Release Time/Date.

Right now, the content gets uploaded when ever the Devs feels to, but setting a Time/Date for when the Achievements will go online could open the door for a more organized way to handle new Set Releases or Revision Updates.

**Comment by kdecks** [comment]
As I see this, it is ultimately superfluous. Without any implementation a dev can do this by uploading during the time of their choosing.
